frontend/public/components/create-yaml.tsx

```diff
@@ -11,7 +11,7 @@ import { ErrorPage404 } from './error';
 import { ClusterServiceVersionModel } from '../models';
 
 export const CreateYAML = connectToPlural((props: CreateYAMLProps) => {
-  const {match, kindsInFlight, kindObj} = props;
+  const {match, kindsInFlight, kindObj, showHeader = true} = props;
```

Comment:
Booleans that default to true are confusing.  I suggest `hideHeader` and default to false